recipes/kissfft/all/conanfile.py

```diff
@@ -0,0 +1,75 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class KissfftConan(ConanFile):
+    name = "kissfft"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/mborgerding/kissfft"
+    description = "a Fast Fourier Transform (FFT) library that tries to Keep it Simple, Stupid"
+    topics = ("conan", "fft", "kiss", "frequency-domain", "fast-fourier-transform")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "datatype": ["float", "double", "int16_t", "int32_t", "simd"],
+               "use_alloca": [True, False]}
+    default_options = {"shared": False,
+                       "fPIC": True,
+                       "datatype": "float",
+                       "use_alloca": False}
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake",
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "_build_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.settings.os == "Windows":
```

Comment:
```suggestion
        if self.options.shared:
```

Reply:
nope, the hook asks to remove fPIC options for Windows, that's a common practice

Reply:
you have the same code twice in two different methods, I thought that was a little strange

Reply:
I think I requested it, but it was a mistake. We want to remove fPIC when building shared. Thank you @prince-chrismc